File: 2025-09-11/reverseSentence.js
function reverseSentence(sentence) {
  
  return sentence
    .split(" ")
    .filter(word => word != '')
    .reverse()
    .join(" ");
}

console.log(reverseSentence("world hello"));
console.log(reverseSentence("push commit git"));
console.log(reverseSentence("npm  install  sudo"));
console.log(reverseSentence("import    default   function  export"));

File: 2025-09/2025-09-11/reverse_sentence.py
def reverse_sentence(sentence):
    
    return " ".join(list(filter(lambda word: word != '', sentence.split(" ")))[::-1])

print(reverse_sentence("world hello"))
print(reverse_sentence("push commit git"))
print(reverse_sentence("npm  install  sudo"))
print(reverse_sentence("import    default   function  export"))